tests/VirtoCommerce.CatalogModule.Tests/CategoryIteratorTests.cs

```diff
@@ -0,0 +1,258 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoFixture;
+using Moq;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.CatalogModule.Core.Model.Search;
+using VirtoCommerce.CatalogModule.Core.Search;
+using VirtoCommerce.CatalogModule.Data.Repositories;
+using VirtoCommerce.CatalogModule.Data.Services;
+using VirtoCommerce.Platform.Core.Settings;
+using Xunit;
+
+namespace VirtoCommerce.CatalogModule.Tests
+{
+    public class CategoryIteratorTests
+    {
+        private readonly Fixture _fixture = new();
```

Comment:
Unavailable feature in .net core 3.1. 
```suggestion
        private readonly Fixture _fixture = new Fixture();
```